Program.cs

```diff
@@ -325,10 +325,29 @@ static void Setup(string username = "")
 			var password = Console.ReadLine();
 
 			UserLogin login = new UserLogin(username, password);
-			Console.Write($"Logging in {username}... ");
-			LoginResult loginResult = login.DoLogin();
-			Console.WriteLine(loginResult);
-			if (!login.LoggedIn) return;
+			string emailCode = null, twoFactorCode = null;
+			while (true)
+			{
+				login.EmailCode = emailCode;
+				login.TwoFactorCode = twoFactorCode;
+				Console.Write($"Logging in {username}... ");
+				LoginResult loginResult = login.DoLogin();
+				Console.WriteLine(loginResult);
+				if (loginResult == LoginResult.NeedEmail)
+				{
+					Console.Write("Email code: ");
+					emailCode = Console.ReadLine();
+					continue;
+				}
+				if (loginResult == LoginResult.Need2FA)
```

Comment:
Since the `if` in preceeded by a `continue`, it would make no difference, so this is a stylistic change, correct?